nomad/pool.go

```diff
@@ -373,6 +374,30 @@ func (p *ConnPool) RPC(region string, addr net.Addr, version int, method string,
 	return nil
 }
 
+// PingNomadServer sends a Status.Ping message to the specified server and
+// returns true if healthy, false if an error occurred
+func (p *ConnPool) PingNomadServer(region string, apiMajorVersion int, s *rpcproxy.ServerEndpoint) (bool, error) {
+	// Get a usable client
+	conn, sc, err := p.getClient(region, s.Addr, apiMajorVersion)
+	if err != nil {
+		return false, err
+	}
+
+	// Make the RPC call
+	var out struct{}
+	err = msgpackrpc.CallWithCodec(sc.codec, "Status.Ping", struct{}{}, &out)
```

Comment:
It's there: https://github.com/hashicorp/nomad/blob/master/nomad/status_endpoint.go#L27